### PR TITLE
Updates to dependabot 

### PR DESCRIPTION
**Pull request by mauriciovasquezbernal**
-  dependabot: Update other packages dependencies too
-  dependabot: use dependency-type: all

### CODE REVIEW

**Review by flyth — APPROVED** [review]
LGTM from inspection

I think it would be better to use the same schedule as for the main repo, though. That way, whenever there's an update, you can check the update and ack them for all at once rather than having to look into what changed again at the weekly time slot (or even someone else has to look into it again).

Would be great if they could be combined like you did in #2916... but that seems to be on another config level.

**Comment by mauriciovasquezbernal** [comment]
> Would be great if they could be combined like you did in https://github.com/inspektor-gadget/inspektor-gadget/pull/2916... but that seems to be on another config level.

It seems not to be supported right now -> https://github.com/dependabot/dependabot-core/issues/1595 :(

**Comment by mauriciovasquezbernal** [comment]
> I think it would be better to use the same schedule as for the main repo, though.

Makes sense to me, the point is, use daily or weekly for all of them. @eiffel-fl do we really need to keep using daily? I asked the same in https://github.com/inspektor-gadget/inspektor-gadget/pull/1331#issuecomment-1436677585, now that I think better, CVEs will trigger security updates that aren't related to the cadence of the dependency update. 

I see many projects are using weekly and event montly:
- https://github.com/containerd/containerd/blob/main/.github/dependabot.yml
- https://github.com/etcd-io/etcd/blob/main/.github/dependabot.yml
- https://github.com/prometheus/prometheus/blob/main/.github/dependabot.yml
- https://github.com/kubernetes/minikube/blob/master/.github/dependabot.yml

File: .github/dependabot.yml [hunk 1]
```diff
@@ -9,6 +9,15 @@ updates:
       timezone: "Europe/Paris"
     commit-message:
       prefix: "ci:"
+  - package-ecosystem: "docker"
+    directory: "/Dockerfiles"
```
[the remark this answers is not in the document]
Comment (mauriciovasquezbernal) [reply .github/dependabot.yml]: I don't know, it was added in #1331 (this PR was just changing the order). I suppose it doesn't have any effect for us as we're using latest.

**Pull request merged**